src/test/java/ServiceObjects/HomePage.java

```diff
@@ -48,4 +50,32 @@ public void checkSearch(String searchTerm) {
         response.then().assertThat().statusCode(200);
         Assertions.assertEquals("Search results",xmlParseByAttribute("class","h2"));
     }
+
+    public void login(String userName, String password) {
+
+
+        getCookiesFilter();
+
+        Response response = httpRequest
+                .contentType("multipart/form-data")
+                .queryParam("controller", "authentication")
+                .queryParam("back", "my-account")
+                .accept("text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+                .redirects().follow(true)
+                .multiPart("token", "8b3ee4f1cba54a1031be9bb7879e9dad")
+                .multiPart("back", "my-account")
+                .multiPart("email", "test@123.com")
+                .multiPart("password", "test123")
+                .multiPart("submitLogin", "1")
+                .when()
+                .post();
+
+        response.getBody().prettyPrint();
+        System.out.println(response.getStatusCode());
+        Response response1 = httpRequest.get().andReturn();
+        response1.getBody().prettyPrint();
+        System.out.println();
```

Comment:
unnecessary prints, should be able to do response = httpRequest.get().andReturn() instead of declaring a 2nd response, since you're finished with 1st response from the post. Then can add in an assertion like:
Assertions.assertEquals("My account", getTitleOfPage())